### DISCUSSION TITLE
Choosing MetaMask when using <InjectedConnector>

### DISCUSSION THREAD

**Discussion by ghost** [wevm/wagmi#1669] [Q&A]
Is it possible to choose exactly MetaMask wallet by click on the button, if two wallets are installed at once (MetaMask and Coinbase Wallet) ?

**Comment by tmm** [comment]
Check out the [`MetaMaskConnector`](https://wagmi.sh/react/connectors/metaMask) or [`InjectedConnector.getProvider`](https://wagmi.sh/react/connectors/injected#getprovider) option.

**Reply by ghost** [reply]
I need exactly to use `InjectedConnector`.
Somehow it is possible to determine exactly MetaMask?
